src/Avalonia.Base/Rendering/Composition/Compositor.cs

```diff
@@ -191,8 +192,8 @@ static void SerializeServerJobs(BatchStreamWriter writer, List<Action> list, obj
                 SerializeServerJobs(writer, _pendingServerCompositorPostTargetJobs, ServerCompositor.RenderThreadPostTargetJobsStartMarker,
                     ServerCompositor.RenderThreadPostTargetJobsEndMarker);
             }
-            
-            _nextCommit.CommittedAt = Server.Clock.Elapsed;
+
+            _nextCommit.CommittedAt = Server.ServerNow;
```

Comment:
@kekekeks this line is the only remaining questionable change.
Previously `Server.Clock.Elapsed` would return actual `right now` elapsed time.
But now it depends whether `UpdateServerTime` was called before that.

Reply:
It was meant to somewhat synchronize animation start time with the UI thread. 
We probably want to share the same time provider between MediaContext and Compositor and use the UI animation time snapshot used for preparing the current frame.
For now the property should just use the current time.